plugins/remotefortressreader/building_reader.cpp

```diff
@@ -316,7 +316,8 @@ void CopyBuilding(int buildingIndex, RemoteFortressReader::BuildingInstance * re
     material->set_mat_index(local_build->mat_index);
 
     remote_build->set_building_flags(local_build->flags.whole);
-    remote_build->set_is_room(local_build->is_room);
+    //FIXME: Figure out what to replace this with.
+    //remote_build->set_is_room(local_build->is_room);
```

Comment:
There's a new `building->relations` list that indicates which civzones ("rooms") this building is associated with

Reply:
The only thing RFR was using this for was for things that were set as rooms but didn't have the room extents set. There's nothing that does that anymore, so I'll be removing the check entirely.